src/Npgsql/MultiplexingDataSource.cs

```diff
@@ -353,8 +353,6 @@ static void CompleteWrite(NpgsqlConnector connector, ref MultiplexingStats stats
             // This will allow the connector to be returned to the pool by its read loop, and also to be selected
             // for over-capacity write.
             connector.FlagAsWritableForMultiplexing();
-
-            NpgsqlEventSource.Log.MultiplexingBatchSent(stats.NumCommands, stats.Stopwatch);
```

Comment:
Dropping the multiplexing metrics for now, because (a) we have various metric issues with multiplexing anyway, and (b) nobody cares :wink: 

We should come back to multiplexing metrics in a separate pass, to understand how everything maps to that world.